modules/mobile/spec/workers/weekly_maintenance_window_logger_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'date'
+
+RSpec.describe Mobile::V0::WeeklyMaintenanceWindowLogger, type: :job do
+  after { Timecop.return }
+
+  describe '#perform' do
+    before do
+      FactoryBot.create(:mobile_maintenance_evss)
+      FactoryBot.create(:mobile_maintenance_mpi)
+      FactoryBot.create(:mobile_maintenance_dslogon)
+    end
+
+    context 'When maintenance windows have been created within the last week' do
+      before do
+        Timecop.freeze(Time.zone.parse('2021-05-29 21:33:39'))
+      end
+
+      it 'Logs maintenance windows' do
+        expect(Rails.logger).to receive(:info).with('Mobile - Maintenance Windows', anything)
+        described_class.new.perform
+      end
+    end
+
+    context 'When maintenance windows have NOT been created within the last week' do
```

Comment:
FWIW, you could do this all in one test by overriding the factory girl defaults, like:
```
      FactoryBot.create(:mobile_maintenance_evss, created_at: 1.day.ago)
      FactoryBot.create(:mobile_maintenance_mpi, created_at: 3.days.ago)
      FactoryBot.create(:mobile_maintenance_dslogon, created_at: 2.weeks.ago)
```
And confirming that only the first two show up. No need for changes though.

Reply:
I was having a lot of trouble checking the exact values of the logger parameters because it puts the dates in a weird date format that I couldn't reproduce on the spec side. So it was easier just to separate it into it hits the logger or doesn't.